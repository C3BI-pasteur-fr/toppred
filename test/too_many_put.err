Warning: too: too many putative segments to calculate best topologies
