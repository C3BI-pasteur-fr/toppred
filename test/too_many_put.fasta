>too many putative segments to calculate all topologies
RRRRRRRVAKLFADAAAAAAASFISPYTRRRRRRR
RRRRRRRVAKLFADAGLVCITSFISPYTRRRRRR
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRR
RVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVR
VRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRR
RRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV
RRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRV