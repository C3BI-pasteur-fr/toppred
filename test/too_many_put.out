Algorithm specific parameters: 

Full window size : 21
Core window size : 11
Wedge window size: 5
Using hydrophobicity file: GES-scale

Cutoff for certain transmembrane segments: 1.00
Cutoff for putative transmembrane segments: 0.60
Critical distance between 2 transmembrane segments: 2

Critical loop length:  60

Kingdom: procaryote

Using cyt/ext file: CYTEXT-scale


Sequence : too  (867 res)
RRRRRRRVAKLFADAAAAAAASFISPYTRRRRRRRRRRRRRRVAKLFADAGLVCITSFIS
PYTRRRRRRRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKL
FADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRR
RRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGL
VCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVR
RRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSF
ISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRR
VAKLFADAGLVCITSFISPYTRRRRRRRVRVRVRRRRRRRVAKLFADAGLVCITSFISPY
TRRRRRRRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFA
DAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRR
VRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVC
ITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRR
RRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRVRRRRRRRVAKLFADAGLVCITSFIS
PYTRRRRRRRVRVRVRRRRRRRVAKLFADAGLVCITSFISPYTRRRRRRRVRRRRRRRVA
KLFADAGLVCITSFISPYTRRRRRRRV

Found: 23 segments

Candidate membrane-spanning segments:

 Helix Begin - End   Score Certainity
     1     8 - 28    0.767 Putative
     2    43 - 63    0.960 Putative
     3    79 - 99    0.960 Putative
     4   117 - 137   0.960 Putative
     5   155 - 175   0.960 Putative
     6   193 - 213   0.960 Putative
     7   231 - 251   0.960 Putative
     8   269 - 289   0.960 Putative
     9   307 - 327   0.960 Putative
    10   345 - 365   0.960 Putative
    11   383 - 403   0.960 Putative
    12   421 - 441   0.960 Putative
    13   461 - 481   0.960 Putative
    14   497 - 517   0.960 Putative
    15   535 - 555   0.960 Putative
    16   573 - 593   0.960 Putative
    17   611 - 631   0.960 Putative
    18   649 - 669   0.960 Putative
    19   687 - 707   0.960 Putative
    20   725 - 745   0.960 Putative
    21   763 - 783   0.960 Putative
    22   803 - 823   0.960 Putative
    23   839 - 859   0.960 Putative
